src/components/Admin/Authorization/EditCommunityCredentials.tsx

```diff
@@ -79,16 +67,17 @@ export const EditCredentials: FC<EditCredentialsProps> = ({ parentMembers, crede
     });
   };
 
-  const availableMembers = useMemo(() => {
-    return parentMembers.filter(p => members.findIndex(m => m.id === p.id) < 0);
-  }, [parentMembers, data]);
-
-  if (loadingMembers) {
-    return <Loading />;
-  }
+  const { available = [], current = [], loading } = useAvailableMembers(credential, resourceId, parentMembers);
```

Comment:
In my opinion returning an empty array will be better.

Reply:
Actually empty arrays are returned, I forgot to remove the default values at some point.